tests/main/ubuntu-core-services/task.yaml

```diff
@@ -0,0 +1,17 @@
+summary: Ensure all services on Core are active
+
+systems: [ubuntu-core-*]
+
+execute: |
+    echo "Ensure one-shot services are working"
+    for oneshot in snapd.autoimport.service snapd.sshd-keygen.service; do
```

Comment:
is it possible to do for each service > systemctl is-active < servicename > ?

Reply:
I think the point is precisely that these are not active because they are oneshot.